src/sentry/runner/commands/cleanup.py

```diff
@@ -227,110 +227,39 @@ def is_filtered(model: type[Model]) -> bool:
                 return False
             return model.__name__.lower() not in model_list
 
-        bulk_query_deletes = generate_bulk_query_deletes()
```

Comment:
This call is only needed within `run_bulk_query_deletes`.

<img width="521" height="238" alt="Image" src="https://github.com/user-attachments/assets/2733c0e8-2996-4fda-bfd0-18ae82479aec" />